spyder/plugins/ipythonconsole/plugin.py

```diff
@@ -546,13 +546,6 @@ def norm(text):
                         before_prompt=True)

```

Comment:
Above this line there's a restriction that doesn't allow to call `runcell` when the console is `_reading` (i.e. our old way to detect if it's in debugging mode).

I think that no longer applies if `function='runcell'`, right? Of course, we can't call `debugcell` if the console is debugging, so the above message would also need to be updated.

Reply:
Well if the shell is _reading or _executing, we can't really start a new execution?

Reply:
Let me put in another way: Are you able to run `runcell` by calling directly in `ipdb`? It'd be nice if users could do that as a way to execute multi-line statements while debugging.

Reply:
I enabled runfile and runcell while debugging

Reply:
That's so cool!! Thanks a lot for doing that!